src/main/java/com/cloudbees/hudson/plugins/folder/computed/ComputedFolder.java

```diff
@@ -108,6 +108,12 @@ protected ComputedFolder(ItemGroup parent, String name) {
         init();
     }
 
+    protected ComputedFolder(ItemGroup parent, String name, OrphanedItemStrategy strategy) {
```

Comment:
`orphanedItemStrategy` is commented as `/*almost final*/`, so I guess we don't want to modify it in a setter (not sure what was the reason to add that comment).